packages/@pollyjs/core/src/test-helpers/jasmine.js

```diff
@@ -0,0 +1,176 @@
+import { afterEach, beforeEach } from './lib';
+
+/**
+ * Flag, showing that Polly is active
+ */
+const IS_POLLY_ACTIVE = Symbol('IS_POLLY_ACTIVE');
+
+/**
+ * Flag, showing that proxy test methods are attached
+ */
+const IS_POLLY_ATTACHED = Symbol('IS_POLLY_ATTACHED');
+
+/**
+ * Shared context to keep polly instance and
+ * options for a specific run
+ */
+const pollyContext = {
+  polly: null,
+  options: {}
```

Comment:
Whats the reason here to store the passed in default config?

Reply:
The same as in https://github.com/Netflix/pollyjs/pull/118#discussion_r226853151 but I spotted a bug while implementing integration tests 😅, so this will change a little. Instead of saving options in context when `setupJasmine` is called as it is right now, it should be applied in `beforeAll`, before the specs will start executing, otherwise I'm always creating Polly instance with the options from the latest `setupJasmine` call